modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -537,12 +537,34 @@ Enable the Iceberg integration for the topic. You can choose one of three modes.
 **Values**:
 
 - `key_value`: Creates an Iceberg table with a `Key` column and a `Value` column. Redpanda stores the raw topic content in the `Value` column.
-- `value_schema_id_prefix`: Creates an Iceberg table whose structure matches the Redpanda schema for the topic, with columns corresponding to each field. Redpanda parses the topic values per field and stores them in the corresponding table columns. 
+- `value_schema_id_prefix`: Creates an Iceberg table whose structure matches the Redpanda schema for the topic, with columns corresponding to each field. Redpanda uses the Schema Registry wire format, consisting of the "magic byte" and schema ID encoded in the payload header, to parse the topic values per field and store them in the corresponding table columns.
+- `value_subject_latest`: Creates an Iceberg table whose structure matches the latest schema version in Schema Registry that matches the subject name. This mode is compatible with Avro and Protobuf schemas and is used when you don't produce to the topic using the wire format.
 - `disabled`: Disables writing to an Iceberg table for the topic.
 
+For `value_subject_latet` mode, use the following syntax:
+
+[,bash]
+----
+value_subject_latest[:<key1>=<value1>,<key2>=<value2>]
```

Comment:
Does `value_subject_latest(:(<key>=<value>,)+)?` mean that 0 to N key-value pairs are allowed?